Intersect.Client.Core/MonoGame/Input/MonoInput.cs

```diff
@@ -249,6 +252,27 @@ private void CheckMouseScrollWheel(int scrlVValue, int scrlHValue)
                 new GwenInputMessage(IntersectInput.InputEvent.MouseScroll, p, MouseButton.Middle, Keys.Alt)
             );
 
+            var deltaV = Math.Sign(mMouseVScroll - scrlVValue);
+
+            if ((DateTime.Now - _lastZoomTime).TotalMilliseconds >= ZoomDelayInMilliseconds)
+
+            {
+
+                if ((InputHandler.HoveredControl == null || !InputHandler.HoveredControl.IsVisibleInTree)
```

Comment:
Doesn't this block scrolling on the canvas?

Shouldn't this or group include `(InputHandler.HoveredControl is Canvas canvas && canvas == canvas.Canvas)`?